deploy/helm/sumologic/values.yaml

```diff
@@ -4273,7 +4276,7 @@ metadata:
         extensions:
           - health_check
           - file_storage
-          # - sumologic
```

Comment:
I believe we can leave this comment here

Reply:
If we do, I'd like to add an explanation to it, because without context it's just confusing.

Reply:
Agreed to the fact that this would need a bit more configuration anyway so it's fine to remove this comment.